astroid/filter_statements.py

```diff
@@ -150,14 +150,9 @@ def _filter_stmts(
                     _stmt_parents.append(stmt.parent)
                 # If the if statement is first-level and not within an orelse block
                 # we know that it will be evaluated
-                elif not if_parent.is_orelse:
+                else:
```

Comment:
Could you update the comment here as well?

Reply:
I can delete it, but I don't have anything better to change it to.